Content.Server/Chat/ChatCommands.cs

```diff
@@ -51,14 +60,17 @@ public void Execute(IConsoleShell shell, IPlayerSession player, string[] args)
 
             var action = string.Join(" ", args);
 
-            chat.EntityMe(player.AttachedEntity, action);
+            // As EntitySay, Hacky way of getting the Player's Mind
+            var mindComponent = player.ContentData().Mind;
```

Comment:
As above, this could probably be tidied up a bit if there was getters/setters for the player object